frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.jsx

```diff
@@ -71,16 +66,118 @@ const ScalarPeriod = ({ lines = 2, period, onClick }) => (
   </ScalarTitleContainer>
 );
 
+function PreviousValueComparison({
+  comparison,
+  width,
+  fontFamily,
+  formatOptions,
+}) {
+  const fontSize = "0.875rem";
+
+  const {
+    comparisonType,
+    percentChange,
+    comparisonPeriodStr,
+    prevValue,
+    changeArrow,
+    changeColor,
+    display,
+  } = comparison;
+
+  const arrowIconName = { "↓": "arrow_down", "↑": "arrow_up" }[changeArrow];
```

Comment:
I'd avoid using symbols like `↓` that are not convenient to type and instead just used `arrow_down`. Seems like these symbols themselves are not used anywhere in UI